src/app/components/cohort-charts/chart.service.js

```diff
@@ -4,9 +4,10 @@ angular.module('transmartBaseUi').factory('ChartService',
     ['Restangular', '$q', '$rootScope', '$timeout', 'AlertService', 'DcChartsService', 'GridsterService',
         function (Restangular, $q, $rootScope, $timeout, AlertService, DcChartsService, GridsterService) {
 
+            var filteringWord = null;
```

Comment:
Please rethink not to have this. Because you can get last dropped node/label in the chart service.